start/src/main/java/org/apache/accumulo/start/classloader/vfs/AccumuloVFSClassLoader.java

```diff
@@ -363,7 +363,10 @@ public static void printClassPath(Printer out) {
             break;
         }
 
-        if (classLoader instanceof URLClassLoader) {
+        if (classLoader.getClass().getName().startsWith("jdk.internal")) {
```

Comment:
Could the final else be improved instead of adding this?

Reply:
I want to be explicit about checking for the Java 11 classloaders, and give a specific message about them.

Reply:
I agree the final else could be improved, and collapsed with this case, but I want to merge as-is, rather than make further changes, because the intent here is to handle one specific case, rather than modify any existing cases. In that regard, the tests are passing now, and I don't want to wait to re-run them to ensure that I didn't break any test cases that currently cover that last else clause.